.travis.yml

```diff
@@ -0,0 +1,45 @@
+dist: trusty
+
+cache:
+  apt: true
+  directories:
+    # Cache directory for older Composer versions.
+    - $HOME/.composer/cache/files
+    # Cache directory for more recent Composer versions.
+    - $HOME/.cache/composer/files
+
+language:
+  - php
+
+php:
+  - 5.6
+  - 7.0
+  - 7.1
+  - 7.2
+  - 7.3
+  - "7.4snapshot"
+
+matrix:
+  fast_finish: true
+  include:
+    - php: 7.3
+    - php: 7.2
+    - php: 7.1
+    - php: 7.0
+    - php: 5.6
+
+  allow_failures:
+    # Allow failures for unstable builds.
+    - php: "7.4snapshot"
+
+before_install:
+    - composer install --dev
+
+script:
+  # Lint the PHP files against parse errors.
+  - if find . -path ./vendor -prune -o -path ./bin -prune -o -name "*.php" -exec php -l {} \; | grep "^[Parse error|Fatal error]"; then exit 1; fi
```

Comment:
Is this really necessary? Won't PHPCS catch parse failures?

Reply:
To my knowledge, it doesn't. PHPCS just tokenizes the code line by line and then checks the sniffs against it. This is included in the WPCS travis probably to ensure that the PHP files don't contain any errors which would then affect the tokenization.

Reply:
Ok, so what I'll do is:
- Add PHP-Parallel-Lint to composer
- Add php versions from 5.6 to 7.3
- Lint all the versions but only check CS on PHP7.3 (when the `PHPCS=1`)
- Add `Generic.PHP.Syntax` exclusion to the `phpcs.xml.dist`

Thanks for the suggestions 🙂